File: SampleObjectiveC/SampleObjectiveC/ViewController.m
//
//  ViewController.m
//  SampleObjectiveC
//
//  Created by ichise on 2017/07/31.
//

#import "ViewController.h"
@import MaterialDesignSymbol;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    MaterialDesignSymbol *symbol = [[MaterialDesignSymbol alloc] initWithText:[MaterialDesignIcon home48px] size:30];
    [symbol addAttributeWithAttributeName:NSForegroundColorAttributeName value:[UIColor blackColor]];
    _sampleImageView.image = [symbol imageWithSize:CGSizeMake(30, 30)];
    
    _sampleLabel.font = [MaterialDesignFont fontOfSize:20];
    _sampleLabel.text = [MaterialDesignIcon clear48px];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

@end

File: Sample/Sample/ViewController.swift
//
//  ViewController.swift
//  Sample
//
//  Created by tichise on 2015年5月10日 15/05/10.
//  Copyright (c) 2015年 tichise. All rights reserved.
//

import UIKit
import MaterialDesignSymbol

class ViewController: UIViewController {

    @IBOutlet var sampleImageView: UIImageView!
    @IBOutlet var sampleLabel: UILabel!

    override func viewDidLoad() {
        super.viewDidLoad()

        // UIImage
        self.sampleImageView.image = MaterialDesignSymbol(icon: .viewHeadline48px, size: 30).image()

        // String
        sampleLabel.font = MaterialDesignFont.fontOfSize(20)
        sampleLabel.text = MaterialDesignIconEnum.clear48px.rawValue
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
}
